### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ahellander**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Describe your changes here to communicate to the maintainers why we should accept this pull request. 
If it fixes a bug or resolves a feature request, be sure to include a link to that issue.

## Types of changes

What types of changes does your code introduce to FEDn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [ ] New feature
- [x] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/fedn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [x] I have updated the related documentation (if necessary) 
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,15 +52,15 @@ Copy the settings config file for the reducer, 'config/settings-reducer.yaml.tem
 Make sure that you can access the Reducer UI at https://localhost:8090. 
 
 ````bash 
-docker-compose -f config/reducer.yaml -f config/private-network.yaml up 
+docker-compose -f config/settings-reducer.yaml -f config/private-network.yaml up
```
Comment (Wrede) [inline README.md:55]: This is not correct? settings-reducer.yaml is used by fedn not docker-compose

File: README.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ Copy the settings config file for the reducer, 'config/settings-reducer.yaml.tem
...
+docker-compose -f config/settings-reducer.yaml -f config/private-network.yaml up 
 ````
 
 4. Start a combiner  
 
 Copy the settings config file for the reducer, 'config/settings-combiner.yaml.template' to 'config/settings-combiner.yaml'. You do not need to make any changes to this file to run the sandbox. To start the combiner service and attach it to the reducer:
 
 ````bash 
-docker-compose -f config/combiner.yaml -f config/private-network.yaml up 
+docker-compose -f config/settings-combiner.yaml -f config/private-network.yaml up
```
Comment (Wrede) [inline README.md:63]: This is not correct? settings-combiner.yaml is used by fedn not docker-compose

**Review by Wrede — CHANGES_REQUESTED** [review]
The settings file are not docker-compose files

File: README.md (hunk 2 above)
Comment (ahellander) [reply README.md:63]: Omg, yes. Closing.

**Pull request closed without merging**